src/liteclient.cc

```diff
@@ -34,6 +34,18 @@ LiteClient::LiteClient(Config& config_in, const AppEngine::Ptr& app_engine, bool
     }
   }
 
+  // figure out the Docker Registry Auth creds endpoint
+  const auto& repo_endpoint = config.uptane.repo_server;
+  std::string auth_creds_endpoint = Docker::RegistryClient::DefAuthCredsEndpoint;
+  if (!repo_endpoint.empty()) {
+    auto endpoint_pos = repo_endpoint.rfind('/');
+    if (endpoint_pos != std::string::npos) {
+      auth_creds_endpoint = repo_endpoint.substr(0, endpoint_pos);
+      auth_creds_endpoint.append("/hub-creds/");
+    }
+  }
+  raw["hub_auth_creds_endpoint"] = auth_creds_endpoint;
```

Comment:
thanks for bubbling this up to the lite-client. It used to bother me being buried so deep down in the code.